app/src/main/java/nu/yona/app/api/manager/network/BaseImpl.java

```diff
@@ -49,9 +48,13 @@ public Response intercept(Chain chain) throws IOException {
             chain.request().newBuilder().addHeader(NetworkConstant.CONTENT_TYPE, "application/json");
             if (NetworkUtils.isOnline(YonaApplication.getAppContext())) {
                 chain.request().newBuilder().addHeader("Cache-Control", "only-if-cached").build();
-            } else if (request.method().equalsIgnoreCase("GET") && !request.cacheControl().noCache()) {
+            } /*else if (request.method().equalsIgnoreCase("GET") && !request.cacheControl().noCache()) {
```

Comment:
why we have commented out  this code ? 
CacheControl not required ?

Reply:
yeah, that's no more require. Removed unused code.